.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -39,24 +45,83 @@ If any further question about AME onboarding or validation tools, please view th
+- [ ] Check this box if any of the following apply to the PR so that label “WaitForARMFeedback” will be added automatically to begin ARM API Review. Failure to comply may result in delays to the manifest.
   - Adding a new service
   - Adding new API(s)
   - Adding a new API version
-    -[ ] To review changes efficiently, ensure you are using OpenAPIHub to initialize the PR for adding a new version. More details, refer to the [wiki](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version).
+    -[ ] To review changes efficiently, ensure you copy the existing version into the new directory structure for first commit and then push new changes, including version updates, in separate commits.
 
 - [ ] Ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.
 
 - [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 
 
 ### Breaking Change Review Checklist 
-If any of the following scenarios apply to the PR, request approval from the Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 
-- [ ] Removing API(s) in a stable version
-- [ ] Removing properties in a stable version
-- [ ] Removing API version(s) in a stable version
-- [ ] Updating API in a stable or public preview version with Breaking Change Validation errors
-- [ ] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))
-
-**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).
+If you have any breaking changes as defined in the [Breaking Change Policy](http://aka.ms/bcforapi), request approval from the Breaking Change Review Board.
+  
+**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Additional details on the process and office hours are on the [Breaking Change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).
```

Comment:
Do we really want to link to internal only documentation here?

Reply:
@JeffreyRichter are you okay with using these links or do you find it problematic?

Reply:
I'm fine with it as long as external people can't access these docs.
We need to cater to our internal teams as best as possible.

Reply:
@weshaggard it seems these links would work as we need them to - to reach internal folks only.

Reply:
@JeffreyRichter is there any reason these cannot or shouldn't be in our public specs repo?

Reply:
Yes, the http://aka.ms/bcforapi PDF file has employee names in it related to legal and marketing and who signs off on breaking changes.

The wiki has internal comments/discussion about proposed changes to policies which must not be external.

Reply:
OK so there might be some things we need to protect but for an open source project there are a lot of links that are going to 404 in this common PR template which is not a great experience.  As a potential future improvement, it would be good to separate these into public and private and keep the private links to a minimum (ideally no more than 1).

Reply:
Hi Wes, I agree with you.  We can prioritize the efforts when we working on to support 3rd party resource provider.